datalad/local/addurls.py

```diff
@@ -1142,6 +1274,16 @@ class Addurls(Interface):
             action="store_true",
             doc="""drop files after adding to annex""",
         ),
+        on_collision=Parameter(
+            args=("--on-collision",),
+            constraints=EnsureChoice("error", "error-if-different",
+                                     "take-first", "take-last"),
+            doc="""What to do when more than one row produces the same file
```

Comment:
I was about to say that "we always start lower-case!", but that is apparently not the case :thinking: 

```sh
% git grep 'doc="""[a-z]' | wc -l
185
% git grep 'doc="""[A-Z]' | wc -l
96
```

Reply:
> ```shell
> % git grep 'doc="""[A-Z]' | wc -l
> 96
> ```

Relieved to see that at least I'm not responsible for _all_ of those.